### PR TITLE
chore: finalize projen monocdk deps

### PR DESCRIPTION
**Pull request by voho**
Finalizes some CDK dependencies and version.

### CODE REVIEW

File: .projenrc.js [hunk 1]
```diff
@@ -56,4 +63,7 @@ project.deps.removeDependency("@aws-cdk/core", DependencyType.RUNTIME);
 project.deps.removeDependency("@aws-cdk/assert");
 project.deps.removeDependency("@aws-cdk/assertions");
 project.deps.removeDependency("monocdk", DependencyType.RUNTIME);
-project.synth();
\ No newline at end of file
+project.deps.removeDependency("constructs", DependencyType.RUNTIME);
+project.deps.addDependency("monocdk@1.123.0", DependencyType.DEVENV);
```
Comment (echeung-amzn) [inline .projenrc.js:67]: Can you extract the version out as a constant?

File: package.json [hunk 2]
```diff
@@ -37,30 +37,35 @@
     "organization": false
   },
   "devDependencies": {
-    "@monocdk-experiment/assert": "^1.123.0",
+    "@monocdk-experiment/assert": "1.123.0",
     "@types/jest": "^27.4.1",
     "@types/node": "^12",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",
+    "constructs": "3.3.69",
```
Comment (echeung-amzn) [inline package.json:45]: How did this end up as 3.3.69 if you specified 3.2.27 in the `.projenrc.js`?

File: package.json [hunk 3]
```diff
@@ -37,30 +37,35 @@
...
+    "prettier": "^2.5.1",
     "projen": "^0.52.44",
     "standard-version": "^9",
     "ts-jest": "^27.1.3",
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
     "constructs": "^3.2.27",
-    "monocdk": "^1.123.0"
+    "monocdk": "1.123.0"
```
Comment (echeung-amzn) [inline package.json:68]: This shouldn't have changed, should it?

**Review by echeung-amzn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (voho) [reply package.json:45]: Now it should be consistent - used the newer version, which is the same as in the current internal library.

File: .projenrc.js (hunk 1 above)
Comment (voho) [reply .projenrc.js]: done

File: package.json (hunk 3 above)
Comment (voho) [reply package.json]: Now ti should be fine, I added some more manual dep changes.

File: .projenrc.js [hunk 4]
```diff
@@ -1,53 +1,62 @@
-const { awscdk, DependencyType} = require('projen');
+const { awscdk, DependencyType } = require("projen");
+const monocdkVersion = "1.123.0";
+const constructsVersion = "3.3.69";
```
Comment (echeung-amzn) [inline .projenrc.js:3]:
```suggestion
const CDK_VERSION = "1.123.0";
const CONSTRUCTS_VERSION = "3.3.69";
```

File: .projenrc.js [hunk 5]
```diff
@@ -1,53 +1,62 @@
...
-  authorAddress: 'monitoring-cdk-constructs@amazon.com',
-  cdkVersion: '1.123.0',
-  defaultReleaseBranch: 'main',
-  name: 'cdk-monitoring-constructs',
-  repositoryUrl: 'git@github.com:cdklabs/cdk-monitoring-constructs.git',
-  cdkDependencies: ['monocdk'],
-  cdkTestDependencies: ['@monocdk-experiment/assert'],
+  author: "CDK Monitoring Constructs Team",
+  authorAddress: "monitoring-cdk-constructs@amazon.com",
+  cdkVersion: monocdkVersion,
```
Comment (echeung-amzn) [inline .projenrc.js:7]:
```suggestion
  cdkVersion: CDK_VERSION,
```

File: .projenrc.js [hunk 6]
```diff
@@ -56,4 +65,15 @@ project.deps.removeDependency("@aws-cdk/core", DependencyType.RUNTIME);
...
+project.deps.addDependency("monocdk@" + monocdkVersion, DependencyType.DEVENV);
+project.deps.addDependency(
+  "constructs@" + constructsVersion,
+  DependencyType.DEVENV
+);
+project.deps.addDependency("monocdk@^" + monocdkVersion, DependencyType.PEER);
+project.deps.addDependency(
+  "constructs@^" + constructsVersion,
+  DependencyType.PEER
+);
```
Comment (echeung-amzn) [inline .projenrc.js:78]:
```suggestion
project.deps.addDependency(`monocdk@${CDK_VERSION}`, DependencyType.DEVENV);
project.deps.addDependency(
  `constructs@${CONSTRUCTS_VERSION}`,
  DependencyType.DEVENV
);
project.deps.addDependency(`monocdk@^${CDK_VERSION}`, DependencyType.PEER);
project.deps.addDependency(
  `constructs@^${CONSTRUCTS_VERSION}`,
  DependencyType.PEER
);
```

**Pull request merged**